### ISSUE TITLE
Interval query

### ISSUE THREAD

**Issue by nshresthan** [elastic/elasticsearch-dsl-py#1328]
Does elasticsearch_dsl already implement "interval" query as in 
https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-intervals-query.html

I tried with the code: 

```
s = s.query("intervals", query=searchWord,
                max_gaps = 1,
                ordered = "True",
                fields=fields2search, 
                type="most_fields", 
                fuzziness="1", operator="AND",
#                 minimum_should_match = "75%",
                
               )
```

but got the error:
UnknownDslObject: DSL class `intervals` does not exist in query.

**Comment by brukatv** [comment]
The intervals query is a new feature in ES 7.0 and as far as I can tell hasn't been implemented yet in the lower level elasticsearch-py api. https://www.elastic.co/guide/en/elasticsearch/reference/7.6/release-notes-7.0.0.html

You can try a quick hack by inheriting from the Match class and changing the name:
```
class IntervalsQuery(Match):
    name = 'intervals'

q = IntervalsQuery(some_field={"query": "foo"})
Search().query(q).execute()
```
Totally winging it. No idea if this will work. I'm on 6.8 myself.

**Comment by alextheinsomniac** [comment]
brukatv's "winging it" solution works !
It saved me having to remove elasticsearch-dsl-py from our code base.
Thanks

The documentation claims "The library provides classes for all Elasticsearch query types."

For the next person using this library until they find that actually it does not support version 7 features when ES is at version 7.8, please implement this simple fix in the code base.
Thanks

**Comment by sethmlarson** [comment]
This was fixed in 7.3.0 in https://github.com/elastic/elasticsearch-dsl-py/pull/1418
